resources/icons/audio-play.svg

```diff
@@ -0,0 +1 @@
+<svg width="20" height="21" viewBox="0 0 20 21" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12.852 6.67c2.724 1.676 4.085 2.514 4.341 3.68.087.398.087.809 0 1.206-.256 1.166-1.617 2.004-4.34 3.68C9.89 17.06 8.41 17.97 7.21 17.62a2.812 2.812 0 0 1-1.095-.611c-.928-.838-.928-2.577-.928-6.055 0-3.478 0-5.217.928-6.054a2.813 2.813 0 0 1 1.095-.612c1.2-.351 2.68.56 5.642 2.383Z" fill="currentColor"/></svg>
```

Comment:
best to remove width and height and only keep the viewBox so it's possible to use different sizes for the image. Same applies to the other one